core/finality-grandpa/src/lib.rs

```diff
@@ -847,22 +847,24 @@ impl<B, E, Block: BlockT<Hash=H256>, RA, PRA> BlockImport<Block>
 		let maybe_change = self.api.runtime_api().grandpa_pending_change(
 			&BlockId::hash(*block.header.parent_hash()),
 			&block.header.digest().clone(),
-		)?;
+		);
+
+        if maybe_change.is_err() {
+            return Err(ConsensusErrorKind::ClientImport.into())
```

Comment:
These `is_err` are a temporary hack to see if I could get the whole thing to compile, I guess some `Into` will need to be implemented instead...